# Retrieve the current extrinsics information from this manager

---
ExtrinsicsInfo info
